src/config/theme/index.js

```diff
@@ -0,0 +1,21 @@
+import {createMuiTheme} from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+  // Add special theming using this reference: https://material-ui.com/customization/default-theme/
+  palette: {
+    primary: {
+      light: '#63ccff',
```

Comment:
nit: either put these in alphabetical order, or an intuitive order? I think main, light, dark, contrast makes sense to me, but feel free to not do this if it already makes sense in current order.